jetbrains-rider/src-201-202/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamProjectGenerator.kt

```diff
@@ -57,10 +53,13 @@ class DotNetSamProjectGenerator(
 
     // TODO: Decouple SamProjectGenerator from the framework wizards so we can re-use its panels
     private val generator = SamProjectGenerator()
-    private val samPanel = SamInitSelectionPanel(generator.wizardFragments) {
+    private val samPanel = SamInitSelectionPanel(
+        generator.wizardFragments,
+        // needed to rerun the validation when the wizard is changed
+        wizardUpdateCallback = { validateData() },
```

Comment:
nit: arguments are named but don't match order they were declared in constructor :(

Reply:
That is funky, fixed